NoSuchMethodError when using WorldEdit API (7.2.0) with FAWE

### Server Implementation

Paper

### Server Version

1.18.2

### Describe the bug

My code interfaces with the WorldEdit API (version 7.2.0), specifically `worldedit-core` and `worldedit-bukkit`.

When using `Region::getBoundingBox()`, a `NoSuchMethodError` is thrown.
Stacktrace:
```accesslog
Caused by: java.lang.NoSuchMethodError: 'com.sk89q.worldedit.regions.CuboidRegion com.sk89q.worldedit.regions.Region.getBoundingBox()'
        at net.abyssalvoid.commandzones.util.WorldEditAPI.getSelection(WorldEditAPI.java:68) ~[commandzones-1.0.0-dev.jar:?]
        at net.abyssalvoid.commandzones.util.SelectionManager.getSelection(SelectionManager.java:31) ~[commandzones-1.0.0-dev.jar:?]
        at net.abyssalvoid.commandzones.command.define.DefineZoneCommand.onExecute(DefineZoneCommand.java:34) ~[commandzones-1.0.0-dev.jar:?]
        at net.abyssalvoid.subcommandlib.PlayerCommand.onExecute(PlayerCommand.java:36) ~[subcommandlib-2.1.0-rc2.jar:?]
        at net.abyssalvoid.subcommandlib.CommandGroup.onExecute(CommandGroup.java:98) ~[subcommandlib-2.1.0-rc2.jar:?]
        at net.abyssalvoid.subcommandlib.CommandGroup.onExecute(CommandGroup.java:98) ~[subcommandlib-2.1.0-rc2.jar:?]
        at net.abyssalvoid.subcommandlib.CommandGroup.onCommand(CommandGroup.java:214) ~[subcommandlib-2.1.0-rc2.jar:?]
        at net.abyssalvoid.subcommandlib.presets.SimpleCommandManager.onCommand(SimpleCommandManager.java:120) ~[subcommandlib-2.1.0-rc2.jar:?]
        at net.abyssalvoid.commandzones.CommandZones.onCommand(CommandZones.java:110) ~[commandzones-1.0.0-dev.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        ... 21 more
```
The code:
![The code, showing where the exception is thrown.](https://user-images.githubusercontent.com/12123847/184793259-3ffb84e1-72c8-481d-8e3b-66aaa43267ac.png)

Maven dependencies:
```xml

		<!-- WorldEdit Support -->
		<dependency>
			<groupId>com.sk89q.worldedit</groupId>
			<artifactId>worldedit-core</artifactId>
			<version>7.2.0</version>
			<scope>provided</scope>
			<exclusions>
				<exclusion>
					<groupId>*</groupId>
					<artifactId>*</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
		<dependency>
			<groupId>com.sk89q.worldedit</groupId>
			<artifactId>worldedit-bukkit</artifactId>
			<version>7.2.0</version>
			<scope>provided</scope>
			<exclusions>
				<exclusion>
					<groupId>*</groupId>
					<artifactId>*</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
```

### To Reproduce

1. Get a `Region`.
2. Attempt to use `Region::getBoundingBox()`
3. `NoSuchMethodError`.

### Expected behaviour

I expected the code to run normally, as I'm using the latest (stable) major & minor version of the WorldEdit API.

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/67f4d8e40b904b0cb8f3058d5b76b490

### Fawe Version

FastAsyncWorldEdit-Bukkit-2.4.4-SNAPSHOT-265

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

### Anything else?
I can confirm the method doesn't exist in the current FAWE API implementation: [com.sk89q.worldedit.regions.Region](https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/main/worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java)

I can also confirm this code works perfectly in [WorldEdit v7.2.12](https://dev.bukkit.org/projects/worldedit/files/3922624).

--- a/worldedit-core/src/main/java/com/sk89q/worldedit/command/ClipboardCommands.java
+++ b/worldedit-core/src/main/java/com/sk89q/worldedit/command/ClipboardCommands.java
@@ -38,6 +38,7 @@ import com.google.common.collect.Lists;
 import com.sk89q.worldedit.EditSession;
 import com.sk89q.worldedit.LocalConfiguration;
 import com.sk89q.worldedit.LocalSession;
+import com.sk89q.worldedit.MaxChangedBlocksException;
 import com.sk89q.worldedit.WorldEdit;
 import com.sk89q.worldedit.WorldEditException;
 import com.sk89q.worldedit.command.util.CommandPermissions;
@@ -104,6 +105,20 @@ import static com.sk89q.worldedit.command.util.Logging.LogMode.REGION;
  */
 @CommandContainer(superTypes = CommandPermissionsConditionGenerator.Registration.class)
 public class ClipboardCommands {
+
+    /**
+     * Throws if the region would allocate a clipboard larger than the block change limit.
+     *
+     * @param region The region to check
+     * @param session The session
+     * @throws MaxChangedBlocksException if the volume exceeds the limit
+     */
+    private void checkRegionBounds(Region region, LocalSession session) throws MaxChangedBlocksException {
+        int limit = session.getBlockChangeLimit();
+        if (region.getBoundingBox().getVolume() > limit) {
+            throw new MaxChangedBlocksException(limit);
+        }
+    }
 
     @Command(
             name = "/copy",

--- a/worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java
+++ b/worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java
@@ -257,6 +257,11 @@ public class CuboidRegion extends AbstractRegion implements FlatRegion {
     }
 
     @Override
+    public CuboidRegion getBoundingBox() {
+        return this;
+    }
+
+    @Override
     public int getMinimumY() {
         return minY;
     }

--- a/worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java
+++ b/worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java
@@ -73,6 +73,15 @@ public interface Region extends Iterable<BlockVector3>, Cloneable, IBatchProcess
     //FAWE end
 
     /**
+     * Get the bounding box of this region as a {@link CuboidRegion}.
+     *
+     * @return the bounding box
+     */
+    default CuboidRegion getBoundingBox() {
+        return new CuboidRegion(getMinimumPoint(), getMaximumPoint());
+    }
+
+    /**
      * Get the center point of a region.
      * Note: Coordinates will not be integers
      * if the corresponding lengths are even.

